src/models/token.rs

```diff
@@ -46,6 +46,28 @@ impl ApiToken {
             last_used_at: self.last_used_at,
         }
     }
+
+    pub fn find_by_api_token_and_revoked(
```

Comment:
This function always gets `false` for `revoked_`. What do you think about leaving this function defined as it was before?

Reply:
Done.